spec/lib/tasks/eu_exit_business_finder_spec.rb

```diff
@@ -0,0 +1,170 @@
+require 'rake'
+require 'rails_helper'
+
+RSpec.describe 'eu_exit_business_finder:update_subscriber_list_titles_and_facet_values' do
+  before :each do
+    Rails.application.load_tasks
+  end
+
+  def mock_facet_values_to_remove(values)
+    allow(EuExitFacetMigrationConfig).to receive(:facet_values_to_remove).and_return(values)
+  end
+
+  def mock_facet_values_to_replace(replacements)
+    allow(EuExitFacetMigrationConfig).to receive(:facet_values_to_replace).and_return(replacements)
+  end
+
+  def mock_facet_value_labels_to_replace(replacements)
+    allow(EuExitFacetMigrationConfig).to receive(:facet_value_labels_to_replace).and_return(replacements)
+  end
+
+  def stub_find_eu_exit_guidance_business_json
+    abs_path_to_fixture_data = Rails.root.join('spec/lib/tasks/fixtures/find-eu-exit-guidance-business.json')
+    stub_request(:get, "#{Plek.find('publishing-api')}/v2/content/42ce66de-04f3-4192-bf31-8394538e0734").
```

Comment:
Here you could use [the test helpers provided by gds-api-adapters](https://github.com/alphagov/gds-api-adapters/blob/d30488d787c98c08702b047f99ce62d42b8a3b1f/lib/gds_api/test_helpers/publishing_api_v2.rb#L369-L378).

I imagine it would look something like:

```suggestion
    stub_publishing_api_has_item(JSON.parse(abs_path_to_fixture_data))
```